src/index.js

```diff
@@ -1 +1,18 @@
-console.log("all is good");
\ No newline at end of file
+import {loadHome} from './homecontent'
+import {loadContact} from './contact'
+import {loadAbout} from './aboutcontent'
+
+
+window.onload = () => {
+    loadHome();
+}
+const home = document.getElementById('home')
+home.addEventListener('click', ()=>{loadHome()});
```

Comment:
No need to wrap loadHome with another anonymous function here. We could just have:
```
home.addEventListener('click', loadHome);
```